### PR TITLE
update doc for using universal redirect

### PR DESCRIPTION
**Pull request by lluunn**
/cc @jlewi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/788)

### CODE REVIEW

File: content/docs/gke/deploy/oauth-setup.md [hunk 1]
```diff
@@ -48,21 +48,14 @@ address to verify the user's identity.
...
-          must have the same value as the deployment name used in the next step 
-          when you deploy Kubeflow from the UI or by running the deployment
-          script. **Deployment name must be 4-20 characters long.**
-          The default value for the deployment name is the `KFAPP` value 
-          used when initializing your Kubeflow app, but you can configure this 
-          with the environment variable `DEPLOYMENT_NAME`.
-        * `<project>` is your GCP project. It must have the same value as used 
-           in the next step when you deploy Kubeflow from the UI or by running 
-           the deployment script.
+        * `<CLIENT_ID>` is the name for your OAuth client ID.
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/oauth-setup.md]: This statement seems wrong, when comparing it to the screenshot:

```
`<CLIENT_ID>` is the name for your OAuth client ID.
```

Based on line 48 above, the name for the OAuth client ID seems to be a user-nominated, human-readable name. But based on the screenshot, it seems to be a generated string. We should either change the description here, or provide a screenshot with a more readable client name if the description is correct.

File: content/docs/gke/deploy/oauth-setup.md [hunk 2]
```diff
@@ -48,21 +48,14 @@ address to verify the user's identity.
    * In the **Name** box enter any name for your OAuth client ID. This is *not*
      the name of your application nor the name of your Kubeflow deployment. It's
      just a way to help you identify the OAuth client ID.
-   * In the **Authorized redirect URIs** box, enter the following:
+   * In the **Authorized redirect URIs** box, enter the following (if not already exists):
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/oauth-setup.md]: Just a small textual fix: change this:

```
if not already exists
```

To this:

```
if it's not already present in the list of authorized redirect URIs
```

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-788--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/oauth-setup/

File: content/docs/gke/deploy/oauth-setup.md [hunk 3]
```diff
@@ -48,21 +48,14 @@ address to verify the user's identity.
...
-          script. **Deployment name must be 4-20 characters long.**
-          The default value for the deployment name is the `KFAPP` value 
-          used when initializing your Kubeflow app, but you can configure this 
-          with the environment variable `DEPLOYMENT_NAME`.
-        * `<project>` is your GCP project. It must have the same value as used 
-           in the next step when you deploy Kubeflow from the UI or by running 
-           the deployment script.
+        * `<CLIENT_ID>` is the name for your OAuth client ID.
+        * Note that the url is not depending on kubeflow deployement and its endpoint, so mulitple
+          kubeflow deployments can share the same oauth client without modifying redirect URIs.
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/oauth-setup.md]: Here's some suggested wording:

```
Note that the URI is not dependent on the Kubeflow deployment or endpoint. Multiple Kubeflow deployments can share the same OAuth client without the need to modify the redirect URIs.
```

File: content/docs/gke/deploy/oauth-setup.md (hunk 1 above)
Comment (lluunn) [reply content/docs/gke/deploy/oauth-setup.md]: good catch, thanks! it's not the "name"

**Comment by sarahmaddox** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**